scripts/dev_deploy_tags.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "Key": "System",
+    "Value": "Authentication"
+  },
+  {
+    "Key": "Environment",
+    "Value": "dev"
+  },
+  {
+    "Key": "Owner",
+    "Value": "di-authentication@digital.cabinet-office.gov.uk"
+  },
+  {
+    "Key": "is-pipeline-deployment",
+    "Value": "False"
+  },
+  {
+    "Key": "repository",
+    "Value": "govuk-one-login/authentication-frontend"
+  }
+]
```

Comment:
This is missing `"Service": "frontend"`

Reply:
`Service` is optional. Maintaining a minimal set, mandatory only https://gds-way.digital.cabinet-office.gov.uk/manuals/aws-tagging.html#mandatory